contrib/re2-cmake/CMakeLists.txt

```diff
@@ -48,6 +50,7 @@ target_compile_definitions (re2_st PRIVATE NDEBUG NO_THREADS re2=re2_st)
 target_include_directories (re2_st PRIVATE .)
 target_include_directories (re2_st SYSTEM PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
 target_include_directories (re2_st SYSTEM BEFORE PUBLIC ${SRC_DIR})
+target_link_libraries (re2_st ch_contrib::abseil_str_format)
```

Comment:
I know it has been like that but the transformation code (l. 55ff) looks very weird and fragile. The reader wonders:
- why transform only the four files given in l. 56? What about the other files?
- is the original build system doing the same stuff or can the library variant (re2 vs. re2_st) be selected via a simple preprocessor define?